pyro_risks/pipeline/score_v0.py

```diff
@@ -71,7 +72,9 @@
 }
 
 
-def prepare_dataset(df, selected_dep=SELECTED_DEP):
+def prepare_dataset(
```

Comment:
See:
https://app.codacy.com/gh/pyronear/pyro-risks/file/55703095833/issues/source?bid=20424053&fileBranchId=23145733#l74

```suggestion
import config as cfg

   df: pd.DataFrame, selected_dep: Optional[List[str]] = None
) -> Tuple[pd.DataFrame, pd.Series]:

     selected_dep = cfg.SELECTED_DEP if selected_dep is None else selected_dep
```